web/auth/auth_config.go

```diff
@@ -0,0 +1,37 @@
+package auth
+
+import (
+	"fmt"
+	"os"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/github"
+)
+
+type AuthConfig struct {
```

Comment:
Does this need to be exported? Suggest it be renamed to just `Config` to avoid stutter: `auth.AuthConfig` vs `auth.Config`.

Reply:
See [comment below](https://github.com/quickfeed/quickfeed/pull/692/files#r921802570). I don't think we need this anymore since we don't want to support multiple providers with one quickfeed server.

Reply:
I have decided that we no longer want to support multiple providers per running quickfeed instance. Hence, we should only need a single auth endpoint. That said, if we need to have a `auth.Config` type, that's fine. But with the current code, I don't see the need. We can just return an `oauth2.Config` object.

Unless you expect a future need in the context of JWTs etc that would require this `auth.Config` type, then I propose to leave it out.